website/content/docs/concepts/domain-model/session-recordings.mdx

```diff
@@ -45,9 +45,25 @@ A session recording belongs to the scope of the [storage bucket][]
 it is stored in. The scope can be either the global scope or an [organization][]
 scope.
 
-## Referenced by
+## Resultant set of policy calculation
 
-- [Storage Bucket][]
+[Storage policies][] codify how long session recordings must be kept and when they should be deleted.
+The resultant set of policy (RSoP) refers to the set of policy attributes that apply to a resource.
+When you apply policies on multiple levels, the results can conflict.
+Boundary evaluates each storage policy attribute individually, starting at the global scope and then evaluating any storage policies assigned to org scopes.
+
+In accordance with the RSoP, Boundary considers the following principles in order:
```

Comment:
I was asked to add some examples of the special edge cases for dealing with 2 policies.